R/bfastlite.R

```diff
@@ -47,4 +47,50 @@ bfastlite <- function(data, formula=response ~ trend + harmon, order = 3,
 }
 
 #' @rdname bfastlite
+#' @export
 bfast0n <- bfastlite
+
+#' Plot the time series and results of BFAST Lite
+#' 
+#' The black line represents the original input data, the green line is the
+#' fitted model, the blue lines are the detected breaks and the whiskers
+#' deone the magnitude (if `magstat` is specified).
```

Comment:
```suggestion
#' The black line represents the original input data,
#' the green line is the fitted model,
#' the blue lines are the detected breaks, and
#' the whiskers deone the magnitude (if `magstat` is specified).
```
I think some logical formatting is welcomed over overall aspect.
Regardless, I am not sure what deone means.
Maybe denotes, although that is a a bit odd as well.